src/routes/dashboard/helpers/channels.ts

```diff
@@ -0,0 +1,75 @@
+import type { Question, Questions } from '../types'
+
+export function sortChannels(questions: Question[]) {
+  const counts = questions.reduce((count, question) => {
+    const key = question.channelName
+    return count[key] ? ++count[key] : (count[key] = 1), count
+  }, {})
+
+  return Object.entries(counts)
+    .map(([channelName, count]) => {
+      return { group: channelName, count }
+    })
+    .sort((a, b) => a.group.localeCompare(b.group))
+}
+
+const percentColors = [
+  { pct: 0, color: { r: 0xff, g: 0x00, b: 0 } },
+  { pct: 50, color: { r: 0xff, g: 0xff, b: 0 } },
+  { pct: 100, color: { r: 0x00, g: 0xff, b: 0 } },
+]
+
+const getColor = (pct: number) => {
+  for (var i = 1; i < percentColors.length - 1; i++) {
```

Comment:
```suggestion
  for (const i = 1; i < percentColors.length - 1; i++) {
```

could probably add this ESLint rule as a warning for future reference https://eslint.org/docs/latest/rules/prefer-const

(I say as a warning because this rule can get very annoying in development)

Reply:
the reason I was using var here is because the value of `i` is referenced outside of the for loop